### DISCUSSION TITLE
My Assets - longevity and storage space

### DISCUSSION THREAD

**Discussion by minim-sheba** [mozilla/hubs#3809] [Spoke Questions] [answered]
Hi all,

I've been developing an audiovisual installation in Hubs and I'm starting to think about launching it, but I had a sudden concern which I haven't been able to find an answer to either here or out there on the Web. I've just been uploading the assets I've been working with to the My Assets area in Spoke, and that's been working fine while I've been working out what I want this to be. However, I can't find any information about whether those files will remain there indefinitely/until I delete them for as long as Hubs exists, or about whether there is any sort of quota for how many things I can upload. Does anyone know the answer to this?

I know just getting some cloud storage space would be the most straightforward answer but pandemic = no work for composer-performers = no money to pay for such a thing so it seemed worthwhile to ask the question!

Thanks,

Caitlin.

**Comment by j-conrad** [comment, marked as answer]
Great question! 
The short answer: Any files you upload to your account in Spoke should remain there as long as you have an account. There are no current plans to remove them. 

Longer answer: If you're depending on Spoke, you'll definitely want to have a backup of your files somewhere (Dropbox, Google Drive, etc.) since there are never any guarantees on server stability, catastrophic failure, etc.
As far as quotas or storage limits go, I'm not exactly sure. For reference though, I've been dropping files into Spoke for a few years now and haven't hit a limit yet.

**Reply by pattersonbl2** [reply]
My understanding is that assets for spoke are saved in ebs volumes in aws if running hubs cloud. by deflat, You have a limit of  8GB but this can be expanded to a larger size in your AWS account.

**Reply by minim-sheba** [reply]
Lovely. Thank you, @j-conrad and @Pattersonbl2 . I did seem to hit a limit because I reached a point where it errored out on me only with larger files (I'm using the publicly available version as no budget to host it myself on AWS), but I discovered that my web hosting comes with a free CDN (Cloudflare) so I've been using that and it's seemed to work pretty well so far. Comforting to know that uploads will remain indefinitely though!
